support/tail.js

```diff
@@ -53,7 +53,7 @@ process.on = function(e, fn){
   if ('uncaughtException' == e) {
     global.onerror = function(err, url, line){
       fn(new Error(err + ' (' + url + ':' + line + ')'));
-      return true;
+      return !mocha.allowUncaught;
```

Comment:
Nope - this is now referring to the `mocha` exposed below on [line 67](https://github.com/mochajs/mocha/pull/1662/files#diff-393a1102677dadf535f6062dbca02af9R67)